tools/check-typo-since

```diff
@@ -32,6 +32,6 @@ check_typo_since() {
 
 case $# in
     0) echo "usage: check-typo-since <git reference>"; exit 2;;
-    1) check_typo_since $1; break;;
+    1) check_typo_since $1;;
```

Comment:
This is not a syntax error ? And this fails silently with `sh` or `dash` but `bash` emits an error. It is probably better to split this commit in a separate PR that I can approve shortly.